### DISCUSSION TITLE
Roadmap?

### DISCUSSION THREAD

**Discussion by braveness23** [jfrog/terraform-provider-artifactory#508] [General] [answered]
Hello,

My company is about to upgrade our JFrog Artifactory to Enterprise+ edition.  As we do this we would like to manage Artifactory, soup to nuts, with Terraform.  I have spent a little bit of time getting familiar with the Artifactory provider and have been able to successfully import some of our existing resources.  However, I see a disparity between the resources that the provider allows me to manage and the resources, features, and functionality available in the console.  

I am pretty new to Artifactory so I cannot give a complete list but some of the things that seem to be missing are projects, layouts, general settings, and Vault authentication provider to name a few.  

I see that the provider is under constant development and is far from stagnant.  Do you have a roadmap that you can share for when features and functionality will be integrated into the provider?

Thanks in advance!

**Comment by alexhung** [comment, marked as answer]
@braveness23 Sorry for late reply. We don't actively monitor the discussions here and GitHub doesn't send notification unless we are tagged/mentioned in the message.

For missing features, projects is handled in a [separate provider](https://github.com/jfrog/terraform-provider-project). Layouts have been [added in August](https://github.com/jfrog/terraform-provider-artifactory/pull/532).

The best way for feature requests is through opening a GitHub issue, as we will get notification email. We will then triage those requests into internal roadmap.

**Reply by alexhung** [reply]
I figured out where to enable notification for discussion (yay) so I'll be receiving emails from now on.
